test/morgan.js

```diff
@@ -822,6 +822,79 @@ describe('morgan()', function () {
         test.write('0')
       })
     })
+
+    describe('a custom', function () {
+      function token (req, res, arg) {
+        return arg
```

Comment:
Can the test be something more real? For one thing, none of the tests here ever actually validate the `req` and `res` are actually those types of objects since this function is so fake. If it was real, then you can read real properties off real requests/responses, which i.m.o. are better real tests that would detect a variety of issues.